ra2ce/graph/network_config_wrapper.py

```diff
@@ -41,21 +42,14 @@
 class NetworkConfigWrapper(ConfigWrapperProtocol):
     ini_file: Path
     config_data: NetworkConfigData
-    graphs: Optional[dict] = None
-    files: dict[str, Path] = {}
+    graphs: Optional[GraphFiles]
```

Comment:
why is it optional if you are already initializing it in line 51? This only make sense when working with dataclasses or not having an `__init__` declaration. As it is right now, it will never be `None` (which is what `Optional` implies).

Reply:
Some references:
https://docs.python.org/3/library/typing.html#typing.Optional
https://peps.python.org/pep-0604/
https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint

Reply:
Will read up on it, thanks.